dev/AppLifecycle/ExtensionContract.h

```diff
@@ -37,6 +37,17 @@ namespace winrt::Microsoft::Windows::AppLifecycle::implementation
 
     inline std::tuple<ExtendedActivationKind, winrt::Windows::Foundation::IInspectable> DecodeActivatedEventArgs(winrt::Windows::Foundation::Uri const& uri)
     {
+        // If the Uri strictly follows uri format from GenerateEncodedLaunchUri, first query parameter is ContractId
```

Comment:
What if this is not true?  If the Uri does not strictly follow the uri format from GenerateEncodedLaunchUri?

Reply:
I see.  I would suggest to add the reasoning why the assumption is safe to make.